### DISCUSSION TITLE
VISIBILITY GROUPS IN FORM

### DISCUSSION THREAD

**Discussion by CACG1022** [opengisch/QField#4047] [Bugs]
Hello, I am currently developing a project in which I created a group box, but I am having problems since I placed an expression so that the group would not be displayed unless the "consecutive" field is equal to 5 but it is the data that Whether in the consecutive field the group is always being displayed.
It should not be displayed because it is not equal to 5, but the field is still displayed. I need it not to show unless it is equal to 5.

attached image:
![ayuda2](https://user-images.githubusercontent.com/118937011/225369612-2b0b5433-4c8f-4db0-b522-e028a4a49da1.png)
[proyecto2.zip](https://github.com/opengisch/QField/files/10982145/proyecto2.zip)

**Comment by CACG1022** [comment]
I also attach the project and thank you for your kind help

**Comment by lucienicolier** [comment]
Hi @CACG1022 ,

I noticed that you are using Shapefiles, can you have a try with a gpkg?

**Comment by CACG1022** [comment]
Hi @lucienicolier , I already tried it, I converted the layers to gpkg and the problem persists

**Comment by VxTedxV** [comment]
Your "consecutiv" field seems to be a text kind, not number, so that your expression can't use "=". Convert your field to 'int' for example to keep this expression or use "like" instead of "=" (with '5' instead of 5)

**Reply by CACG1022** [reply]
Hi @VxTedxV  consecutive field is set to integer
